### PR TITLE
create bit insertion test file and solution file

### PR DESCRIPTION
**Pull request by samedkocyigit**

### CODE REVIEW

File: InterviewQuestions/Chapters/BitInsertions/BitInsertion.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+namespace InterviewQuestions.Chapters.BitInsertion
+{
+    public class BitInsertions
+    {
+        public int BitInsertionSolution(int N,int M, int i, int j)
```
Comment (furkankocyigit) [inline InterviewQuestions/Chapters/BitInsertions/BitInsertion.cs]: Please give appropriate names for arguments. what are those i,j M, and N?
Are i and j startIndex and endIndex?

File: InterviewQuestionsTests/Chapters/BitInsertions/BitInsertions.Tests.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        public void BitInsertionMethod()
+        {
+            BitInsertions test1 = new BitInsertions();
+            int temp =test1.BitInsertionSolution(128,19,2,6);
+            int temp1 =test1.BitInsertionSolution(65,12,1,4);
+
+            Assert.AreEqual(temp,204);
+            Assert.AreEqual(temp1,89);
+        }
+    }
```
Comment (furkankocyigit) [inline InterviewQuestionsTests/Chapters/BitInsertions/BitInsertions.Tests.cs:20]: Could you convert this test to a parametric test and use the following test cases?

[DataRow(128, 19, 2, 6, 204)]
[DataRow(65, 12, 1, 4, 89)]
[DataRow(2560, 7, 3, 5, 2616)]
[DataRow(145, 10, 2, 5, 169)]

File: InterviewQuestionsTests/Chapters/BitInsertions/BitInsertions.Tests.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
+using InterviewQuestions.Chapters.BitInsertion;
+
+namespace InterviewQuestionsTests.Chapters.BitInsertion
+{
+    [TestClass] 
+    public class BitInsertionTest
+    {
+        [TestMethod]
+        public void BitInsertionMethod()
```
Comment (furkankocyigit) [inline InterviewQuestionsTests/Chapters/BitInsertions/BitInsertions.Tests.cs]: could you change this test name to something like "bitInsertionTest" ?

**Review by furkankocyigit — CHANGES_REQUESTED** [review]
Please see my comments.

File: InterviewQuestionsTests/Chapters/BitInsertions/BitInsertions.Tests.cs (hunk 2 above)
Comment (samedkocyigit) [reply InterviewQuestionsTests/Chapters/BitInsertions/BitInsertions.Tests.cs:20]: done, you can look here 7758d34a7b1aa87985bb842eb81d5af158e2e2e9

File: InterviewQuestionsTests/Chapters/BitInsertions/BitInsertions.Tests.cs (hunk 3 above)
Comment (samedkocyigit) [reply InterviewQuestionsTests/Chapters/BitInsertions/BitInsertions.Tests.cs]: done, you can look here 7758d34a7b1aa87985bb842eb81d5af158e2e2e9

File: InterviewQuestions/Chapters/BitInsertions/BitInsertion.cs (hunk 1 above)
Comment (samedkocyigit) [reply InterviewQuestions/Chapters/BitInsertions/BitInsertion.cs]: arguments renamed, you can look here fb2ebc838d9c505ba096e3ef57dbb4cfa4f01e9a

**Review by furkankocyigit — APPROVED** [review]
LGTM!

**Pull request merged**